### ISSUE TITLE
Unexpected crash

### ISSUE THREAD

**Issue by akraa** [codesandbox/codesandbox-client#4085]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 81.0.4044 | Linux x86_64           | d4d6050a6          |

  **Route:**
  https://codesandbox.io/s/lucid-greider-g6fz8?file=/src/App.js:2167-2204

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'username' of undefined
  TypeError: Cannot read property 'username' of undefined
    at https://codesandbox.io/static/js/app.3e22811ea.js:602:1391414
    at Array.map (<anonymous>)
    at Ec (https://codesandbox.io/static/js/app.3e22811ea.js:602:1390936)
    at ua (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1713458)
    at Ka (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1721723)
    at xs (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1759911)
    at ml (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1744297)
    at pl (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1744222)
    at il (https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1741550)
    at https://codesandbox.io/static/js/common.db2eeb52e.chunk.js:1:1699733
  
    in Ec
    in t
    in Unknown
    in div
    in t
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in t
    in styled.div
    in div
    in t
    in elements__Container
    in t
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in t
    in elements__Content
    in r
    in withRouter(r)
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in t
    in l
    in t
  ```

  </details>

**Comment by SaraVieira** [comment]
Hey!

Can you please provide some info on what you were doing when this crash happened?

**Comment by lbogdan** [comment]
Closing this, it should already by fixed in #4176 .
